src/style.css

```diff
@@ -0,0 +1,103 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: normal;
```

Comment:
default values. you don't have to set those

Reply:
The tests in browser are all passed, what is the problem with git?